api/client/client.go

```diff
@@ -49,6 +48,7 @@ import (
 	"google.golang.org/grpc/credentials"
 	ggzip "google.golang.org/grpc/encoding/gzip"
 	"google.golang.org/grpc/keepalive"
+	"google.golang.org/protobuf/types/known/emptypb"
```

Comment:
Changes beyond this line are all IDE/Editor made (alias import to `empty` -> rename import to `emptypb` -> drop alias).

Same for other files.